src/sample/models/Bot.java

```diff
@@ -52,15 +142,19 @@ private void moveTheBot(MoveEnum direction)
         {
             case Up:
```

Comment:
This is actually doing a move, so ```getPossibleMoves``` isn't correct anymore.

Reply:
makes sense. had to look at the actual code for some reason to see the change but yeah we good. silly gitty.